docs/compile.md

```diff
@@ -355,6 +355,7 @@ parameters:
       	namespace: <substitutes_.Release.Namespace>
       	name_template: <namespace_template>
       	release_name: <chart_release_name>
+      kube_version: "1.16"
```

Comment:
nit: needs an extra space

Reply:
I think it was correct. You were probably mislead by the fact that the above lines (but not all) included tab characters. I replaced those by spaces. Hope you don't mind.

Also, I changed the "1.16" to <target_kubernetes_version_string> to harmonize with other params, and moved the example into the description.